examples/methods/plot_kmm_reweighting.py

```diff
@@ -0,0 +1,107 @@
+"""
+Kernel Mean Matching
+====================
+This example illustrates the use of KMM method [1] to correct covariate-shift.
```

Comment:
Small rendering problem in the example

![image](https://github.com/scikit-adaptation/skada/assets/1386873/579c7edc-e334-47d9-8bff-2a70ad6da784)